sky/server/uvicorn.py

```diff
@@ -0,0 +1,73 @@
+"""Uvicorn wrapper for Sky server."""
```

Comment:
```suggestion
"""Uvicorn wrapper for SkyPilot API server."""
```
Would be good to include the benchmark numbers in the docstr as well. : )

Reply:
Sounds good!